Cargo.toml

```diff
@@ -14,9 +14,7 @@ members = [
 	"bridge/primitives/*",
    "evolution-chain/runtime",
    "evolution-chain/node",
-   "evolution-chain/pallets/*",
    "evolution-chain/primitives",
```

Comment:
I think we also need to bring primitives from laos-evolution ?

Reply:
there's no dependency from the evolution pallet and precompiles with primitives of evolution chain